package.json

```diff
@@ -26,7 +26,7 @@
     "db:migration:run": "yarn typeorm -- migration:run  -d ormconfig.ts"
   },
   "dependencies": {
-    "@across-protocol/contracts-v2": "^1.0.0",
+    "@across-protocol/contracts-v2": "^1.0.5",
```

Comment:
This will have to be upgraded soon to 1.0.6 with neew MerkleDistributor contract  https://github.com/across-protocol/contracts-v2/pull/195

Reply:
Ok I will wait until the new version of the package is published.